ddtrace/contrib/unittest/patch.py

```diff
@@ -522,6 +690,20 @@ def _start_test_module_span(instance) -> ddtrace.Span:
     test_module_span.set_tag_str(test.TEST_TYPE, SpanTypes.TEST)
     test_module_span.set_tag_str(test.MODULE, test_module_name)
     test_module_span.set_tag_str(test.MODULE_PATH, _extract_module_file_path(instance))
+    test_module_span.set_tag_str(
+        test.ITR_TEST_CODE_COVERAGE_ENABLED,
+        "true" if _CIVisibility._instance._collect_coverage_enabled else "false",
+    )
+    if _CIVisibility.test_skipping_enabled():
+        test_module_span.set_tag_str(test.ITR_TEST_SKIPPING_ENABLED, "true")
+        test_module_span.set_tag_str(test.ITR_TEST_SKIPPING_TYPE, TEST)
+        test_module_span.set_tag_str(test.ITR_TEST_SKIPPING_TESTS_SKIPPED, "false")
+        test_module_span.set_tag_str(test.ITR_DD_CI_ITR_TESTS_SKIPPED, "false")
+        test_module_span.set_tag_str(test.ITR_FORCED_RUN, "false")
+        test_module_span.set_tag_str(test.ITR_UNSKIPPABLE, "false")
```

Comment:
The duplication of logic between this block and line 651 increases the likelihood of subtle bugs being introduced where one block is changed independently of the other. I recommend deduplicating this code.

Reply:
Good suggestion, should've been addressed now